nncf/openvino/graph/layer_attributes.py

```diff
@@ -67,9 +67,68 @@ def get_const_port_ids(self) -> List[int]:
             return list(self._constant_attributes.keys())
         return []
 
+    def get_backend_agnostic_attributes(self):
+        return self._layer_attributes
+
+
+OV_CONV_METATYPES = [
+    OVConvolutionMetatype,
+    OVDepthwiseConvolutionMetatype,
+    OVGroupConvolutionMetatype,
+    OVConvolutionBackpropDataMetatype,
+    OVGroupConvolutionBackpropDataMetatype,
+]
+
+
+OVConvLayout = List[OVConvLayoutElem]
+
+
+def get_conv_weights_layout_from_node(node: NNCFNode) -> OVConvLayout:
+    layer_attributes = node.layer_attributes
+    port_id = _get_port_id_from_layer_attributes(layer_attributes)
+    return get_conv_weights_layout(
+        ov_metatype=node.metatype, weights_shape=layer_attributes.constant_attributes[port_id]["shape"]
+    )
+
+
+def get_linear_weights_layout_from_node(node: NNCFNode) -> OVConvLayout:
+    layer_attributes = node.layer_attributes
+    port_id = _get_port_id_from_layer_attributes(layer_attributes)
+    constant_layer_attrs = layer_attributes.constant_attributes[port_id]
+    return get_linear_weights_layout(
+        weights_shape=constant_layer_attrs["shape"],
+        transpose=constant_layer_attrs.get("transpose", False),
+        port_id=port_id,
+    )
+
+
+def _get_port_id_from_layer_attributes(layer_attributes):
+    port_ids = list(layer_attributes.constant_attributes.keys())
+    assert len(port_ids) == 1
+    return port_ids[0]
```

Comment:
Is it possible to call this function for `LSTM` or `GRU` nodes?

Reply:
This is axillary function applicable only for Conv and MatMul layers. LSTM and GRU are using `GenericWeightedLayerAttributes` for now